app/src/components/CardGrid.tsx

```diff
@@ -32,10 +33,10 @@ const CardGrid: React.FC<Props> = ({ status, isFetching, data }) => {
             <h2 className="card__headline">{animal.name}</h2>
             <span className="card__type">{animal.type}</span>
             <Animal
-              name={`${animal.animal === "crocodile" ? "alligator" : "tiger"}`}
-              color={`${
-                animal.animal === "crocodile" ? "#7bb853" : "#f0cc55"
-              }`}
+              name={
+                animalIcons[animal.animal as keyof AnimalIconsInterface].name
+              }
+              color={`${animal.animal === "crocodile" ? "#7bb853" : "#f0cc55"}`}
```

Comment:
I have made a suggestion to cleanup this part of the code:
https://github.com/reload/pet-dream-store/pull/9